### ISSUE TITLE
Cannot close object

### ISSUE THREAD

**Issue by cantonc62** [overcuriousity/pdf2epub#5]
Error converting /Users/tonc/Desktop/deneme/hemato.pdf: Mismatch between logits and bboxes. Logits: 56, Bboxes: 59
-> Cannot close object, library is destroyed. This may cause a memory leak!

I get this error and the final form of the epub is only 2 pages instead of 165

**Comment by overcuriousity** [comment]
I encountered this also. Will have a look into it soon.

**Comment by zwx00** [comment]
Yup, same issue.

**Comment by overcuriousity** [comment]
Root cause was in surya's `ordering.py` (reading-order model), used by the previously pinned marker-pdf 0.3.10 — that's where the "Mismatch between logits and bboxes" error came from. marker-pdf is now pinned to 1.10.2, whose surya (0.17.x) removed that module entirely, so the failing code path no longer exists. Please reinstall from current main (`pip install -r requirements.txt`) and reopen if it still reproduces.
